cmake/configure/configure_1_cuda.cmake

```diff
@@ -25,29 +25,31 @@ SET(FEATURE_CUDA_DEPENDS CXX11)
 
 MACRO(FEATURE_CUDA_FIND_EXTERNAL var)
 
-  #
-  # FIXME Restructure this call into a ./modules/FindCUDA.cmake file and
-  # use DEAL_II_PACKAGE_HANDLE
-  #
-  FIND_PACKAGE(CUDA)
-  MARK_AS_ADVANCED(
-    CUDA_HOST_COMPILER
-    CUDA_SDK_ROOT_DIR
-    CUDA_TOOLKIT_ROOT_DIR
-    CUDA_USE_STATIC_CUDA_RUNTIME
-    )
+  IF(${DEAL_II_WITH_CUDA})
```

Comment:
Please remove the `if(${DEAL_II_WITH_CUDA})` statement and simply set
```
SET(DEAL_II_WITH_CUDA FALSE CACHE BOOL "") 
```
at the top of the file, i.e., outside of the macro definition.